### PR TITLE
Ovirt: rename Smoke to Early

### PR DESCRIPTION
**Pull request by eslutsky**
In origin commit 07bef47, the Smoke tests were renamed to Early,
causing the run-minimal-tests() function to fail with:

`error: suite "files" does not contain any tests`

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml [hunk 1]
```diff
@@ -413,6 +413,14 @@ objects:
           ${SSH_PUB_KEY}
         EOF
 
+        function update_image_registry() {
```
Comment (rgolangh) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: Please move this to a separate PR, it's not related to this fix.

Comment (eslutsky) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: Both PRs required for the test to pass.

Comment (eslutsky) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml]: both commits pass minimal tests , removed the registry commit.

**Comment by eslutsky** [comment]
@sdodson , to pass minimal tests it should be merged with https://github.com/openshift/release/pull/7091
